src/benchmarking.rs

```diff
@@ -2,30 +2,106 @@
 
 use super::*;
 
-#[allow(unused)]
-use crate::Pallet;
-use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use crate::{
+    Pallet as PeaqMor,
+    types::{BalanceOf, MorConfig},
+};
+use peaq_pallet_did::Pallet as PeaqDid;
+use frame_benchmarking::{
+    account, benchmarks, impl_benchmark_test_suite
+};
 use frame_system::{Pallet as System, RawOrigin};
+use sp_runtime::traits::Zero;
 
 /// Assert that the last event equals the provided one.
 fn assert_last_event<T: Config>(generic_event: <T as Config>::Event) {
     System::<T>::assert_last_event(generic_event.into());
 }
 
-const CALLER_ACCOUNT_STR: &str = "Peaq";
+
+// Copy & paste constants from mock.rs
+const O_ACCT: &'static str = "Alice"; // Owner
+const U_ACCT: &'static str = "Bob"; // User
+const M_ACCT: &'static str = "RPi001"; // Machine
+const M_ATTR: &[u8] = b"Type";
+const M_VAL: &[u8] = b"RPiMachine";
+const REG_FEE: u128 = 100_000_000_000_000_000u128;
+
```

Comment:
Is this duplication necessary? Why not use the publicly declared constants on `mock.rs`?